x/ibc/20-transfer/types/keys.go

```diff
@@ -18,7 +18,7 @@ const (
 	Version = "ics20-1"
 
 	// PortID that transfer module binds to
-	PortID = "bank"
+	PortID = "transfer"
```

Comment:
double-checking if is this correct?

Reply:
@AdityaSripal ^

Reply:
Yes this is good. I prefer `transfer` to `bank` tbh.